src/main/java/org/pesmypetcare/webservice/dao/medalmanager/MedalDaoImpl.java

```diff
@@ -82,4 +88,20 @@ private void initializeWithCollectionPath() {
         batch = dbCol.batch();
         path = Path.ofCollection(Collections.medals);
     }
+
+    /**
+     * Insert the new medal to all users.
+     * @param medal The new medal we had created before
+     * @throws DatabaseAccessException When the retrieval is interrupted or the execution fails
+     * @throws DocumentException When the document does not exist
+     */
+    private void addMedalsToUsers(MedalEntity medal) throws DatabaseAccessException, DocumentException {
+        path = Path.ofCollection(Collections.used_usernames);
+        List<DocumentSnapshot> allUsers = dbCol.listAllCollectionDocumentSnapshots(path);
+        UserMedalEntity medalEntity  = new UserMedalEntity(medal.getName(), 0., 0.,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a single space to separate non-whitespace characters.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5630992)